package.json

```diff
@@ -37,7 +37,7 @@
     "test:babel-plugins": "yarn workspace @glimmer/vm-babel-plugins test",
     "test:browserstack": "ember test --test-port=7774 --host 127.0.0.1 --config-file=testem-browserstack.js",
     "test:lint": "eslint . --quiet",
-    "test:node": "node bin/run-node-tests.mjs",
```

Comment:
nothing was calling `test:node` prior to this PR, and run-node-tests called global `ember`, which isn't used in CI